### PR TITLE
Rpm group creation

### PR DESCRIPTION
**Pull request by iigorkarpov**
##### Summary
When installing the `netdata` package, the `netdata` user account will now be added to the following groups (only if these groups are present in the system at the time of installation):

`docker nginx varnish haproxy adm nsd proxy squid ceph nobody I2C`

##### Test Plan

Build RPM package on the target system with this patch. Do local install.

Successfully tested on Rocky Linux 8

##### Additional Information
`netdata` user should be the member of some groups to read corresponding log files
<details> <summary>For users: How does this change affect me?</summary>
One should not add `netdata` to these groups manually to let it obtaining the data
</details>

Fixes: https://github.com/netdata/netdata/issues/13090

### CODE REVIEW

File: netdata.spec.in [hunk 1]
```diff
@@ -447,12 +447,20 @@ ${RPM_BUILD_DIR}/%{name}-%{version}/packaging/bundle-ebpf.sh ${RPM_BUILD_DIR}/%{
...
-getent group docker >/dev/null || groupadd -r docker
-getent passwd netdata >/dev/null || \
-  useradd -r -g netdata -G docker -s /sbin/nologin \
-    -d %{contentdir} -c "netdata" netdata
+if ! getent group netdata > /dev/null; then
+  groupadd --system netdata
+fi
+
+if ! getent passwd netdata > /dev/null; then
+  useradd --system -g netdata --home %{contentdir} --no-create-home -s /sbin/nologin -c "Netdata pseudo user" netdata
```
Comment (ilyam8) [inline netdata.spec.in:455]: What dir is `%{contentdir}`? Can you show `getent passwd netdata` after installing?

Comment (iigorkarpov) [reply netdata.spec.in:455]: %{contentdir} is defined in the start of this script. As you can see, it's been used in earlier version of this function. I just used it, though I'm not sure in its usefulness.

Comment (iigorkarpov) [reply netdata.spec.in:455]: What do you mean by 'show after installing'? In the script?

Comment (Ferroin) [reply netdata.spec.in:455]: `/var/lib/netdata` should be the correct home directory in this case.

File: netdata.spec.in [hunk 2]
```diff
@@ -447,12 +447,20 @@ ${RPM_BUILD_DIR}/%{name}-%{version}/packaging/bundle-ebpf.sh ${RPM_BUILD_DIR}/%{
...
+fi
+
+# I am not sure that the corresponding group names are exactly the same as in Debian, but we should start somewhere, shouldn't we?
+for item in docker nginx varnish haproxy adm nsd proxy squid ceph nobody I2C; do
```
Comment (Ferroin) [inline netdata.spec.in:459]: The only three that I think _might_ be different are `adm`, `nsd`, and `I2C`. All the others should be the same.

**Comment by ilyam8** [comment]
> Build RPM package on the target system with this patch. Do local install.

We also need to test upgrading - upgrading should add `netdata` to the groups too.

**Comment by Ferroin** [comment]
> > Build RPM package on the target system with this patch. Do local install.
> 
> We also need to test upgrading - upgrading should add `netdata` to the groups too. We found out that that was the [problem with deb packages](https://github.com/netdata/netdata/pull/13252).

It does indeed need tested, but is likely to be a non-issue (RPM script hooks for packages are much more primitive than in DEB packages, and get run the same way regardless).

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**